dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+import "oaidl.idl";
+import "ocidl.idl";
+
+#include "../PushNotifications-Constants.h"
+
+[object]
+[uuid(60FC21B2-B396-4D49-94F0-7555869FB93C)]
+[pointer_default(unique)]
+interface INotificationsLongRunningPlatform : IUnknown
+{
+    HRESULT RegisterActivator([in] LPCWSTR processName);
+
+    HRESULT UnregisterActivator([in] LPCWSTR processName);
+
+    HRESULT RegisterForegroundActivator([in] LPCWSTR processName);
```

Comment:
Can we remove API signatures that haven't been finalized from this list? I don't think anything other than RegisterApplication is finalized right?